code/Forms/UsedOnTable.php

```diff
@@ -85,8 +85,8 @@ public function usage(HTTPRequest $request)
 
             $usageData[] = [
                 'id' => $userId,
-                'title' => $user->getTitle(),
-                'type' => $user->singular_name(),
+                'title' => $user->getTitle() ?: _t(__CLASS__ . '.UNTITLED', '(Untitled)'),
```

Comment:
This should be `Untitled` rather than `(Untitled)` to match what was agreed [here](https://github.com/dnadesign/silverstripe-elemental/issues/574#issuecomment-675830833)